code/WorkInProgress/Cael_Aislinn/Rust/core_field.dm

```diff
@@ -6,7 +6,7 @@ Deuterium-tritium fusion: 4.5 x 10^7 K
 
 //#DEFINE MAX_STORED_ENERGY (held_plasma.toxins * held_plasma.toxins * SPECIFIC_HEAT_TOXIN)
 
-/obj/effect/rust_em_field
+/obj/machinery/power/rust_em_field
```

Comment:
If I remember correctly, the rad collectors and the EM field being on two separate process lists caused problems with getting the radiation to properly convert into power and be put into the grid.